### PR TITLE
expand time and currencies

### PR DESCRIPTION
**Pull request by Mic92**

### CODE REVIEW

**Comment by erogol** [comment]
Thx for the PR!
Please check CI from here https://travis-ci.org/github/mozilla/TTS
All checks failed 
For some reason, Github does not show Travis anymore.

**Comment by Mic92** [comment]
Another issue is that there is no model released that I can use with master. I don't have GPUs to train my own models. That's why I tested my code against: https://github.com/mozilla/TTS/tree/72a6ac5

**Comment by Mic92** [comment]
I can also help with migrating it over to github actions. Travis now only gives you an hour of test time per month or so. I had other projects where I run into this limitation.

**Comment by Mic92** [comment]
Ok. I see it breaks with python3.6 while I was testing python3.8. What range of python version is your library targeted at?

**Comment by Mic92** [comment]
CI builds now: https://travis-ci.org/github/mozilla/TTS/builds/747899477

**Comment by erogol** [comment]
> I can also help with migrating it over to github actions. Travis now only gives you an hour of test time per month or so. I had other projects where I run into this limitation.

does github actions provide CI? Is it free for open source ?

**Comment by erogol** [comment]
> Ok. I see it breaks with python3.6 while I was testing python3.8. What range of python version is your library targeted at?

we say python>=3.6 in README

**Comment by Mic92** [comment]
> > I can also help with migrating it over to github actions. Travis now only gives you an hour of test time per month or so. I had other projects where I run into this limitation.
> 
> does github actions provide CI? Is it free for open source ?

Yes and Yes, you only pay for private projects after using up your credit.

**Comment by Mic92** [comment]
> > Ok. I see it breaks with python3.6 while I was testing python3.8. What range of python version is your library targeted at?
> 
> we say python>=3.6 in README

Ok. Than I will also try to restore support in CI.

**Comment by erogol** [comment]
Can you rebase this with the latest dev so that we see the CI status?

**Comment by erogol** [comment]
Also maybe it is worth to check something like https://github.com/savoirfairelinux/num2words

**Comment by Mic92** [comment]
num2word does not support times and it also cannot detect currencies on its own https://github.com/savoirfairelinux/num2words#usage

**Comment by Mic92** [comment]
Where is the CI status now?

**Comment by Mic92** [comment]
Locally I get wrong broken test:

```
ImportError while importing test module '/home/joerg/git/TTS/tests/test_glow_tts.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
tests/test_glow_tts.py:10: in <module>
    from TTS.tts.models.glow_tts import GlowTts
TTS/tts/models/glow_tts.py:9: in <module>
    from TTS.tts.layers.glow_tts.monotonic_align import maximum_path, generate_path
TTS/tts/layers/glow_tts/monotonic_align/__init__.py:5: in <module>
    from TTS.tts.layers.glow_tts.monotonic_align.core import maximum_path_c
E   ModuleNotFoundError: No module named 'TTS.tts.layers.glow_tts.monotonic_align.core'
```

However this does not look like something I changed.

**Comment by erogol** [comment]
To see CI you need to rebase. It is on Circle CI.

It looks like you did not install TTS correctly. At least the cython dependency.

**Comment by erogol** [comment]
Can you send a new PR? From now on CI should run for forked PRs as well. Otherwise, I need to find a different CI provider or avoid CI for now. Sorry for the inconvenience :(

File: tests/test_text_processing.py [hunk 1]
```diff
@@ -171,4 +171,4 @@ def test_text2phone():
     gt = "ɹ|iː|s|ə|n|t| |ɹ|ɪ|s|ɜː|tʃ| |æ|t| |h|ɑːɹ|v|ɚ|d| |h|ɐ|z| |ʃ|oʊ|n| |m|ɛ|d|ᵻ|t|eɪ|ɾ|ɪ|ŋ| |f|ɔː|ɹ| |æ|z| |l|ɪ|ɾ|əl| |æ|z| |eɪ|t| |w|iː|k|s| |k|æ|n| |æ|k|tʃ|uː|əl|i| |ɪ|n|k|ɹ|iː|s|,| |ð|ə| |ɡ|ɹ|eɪ| |m|æ|ɾ|ɚ|ɹ| |ɪ|n|ð|ə| |p|ɑːɹ|t|s| |ʌ|v|ð|ə| |b|ɹ|eɪ|n| |ɹ|ɪ|s|p|ɑː|n|s|ə|b|əl| |f|ɔː|ɹ| |ɪ|m|oʊ|ʃ|ə|n|əl| |ɹ|ɛ|ɡ|j|uː|l|eɪ|ʃ|ə|n| |æ|n|d| |l|ɜː|n|ɪ|ŋ|!"
     lang = "en-us"
     ph = text2phone(text, lang)
-    assert gt == ph, f"\n{phonemes} \n vs \n{gt}"
+    assert gt == ph, f"\n{ph} \n vs \n{gt}"
```
Comment (erogol) [inline tests/test_text_processing.py:174]: don't just lazy test. It is better if you create a new function testing specifically cases you try to solve.

Comment (Mic92) [reply tests/test_text_processing.py:174]: I did not changed anything in the test, but the test uses the wrong variable here (`ph` instead of `phonemes`).

**Comment by Mic92** [comment]
See https://github.com/mozilla/TTS/pull/596

**Pull request closed without merging**